File: dbflow/src/androidTest/java/com/raizlabs/android/dbflow/test/customhelper/HelperDatabase.java
package com.raizlabs.android.dbflow.test.customhelper;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Description:
 */
@Database(name = HelperDatabase.NAME, version = HelperDatabase.VERSION,
        sqlHelperClass = CustomOpenHelper.class)
public class HelperDatabase {

    public static final String NAME = "Helper";

    public static final int VERSION = 1;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/customhelper/HelperDatabase.kt
package com.raizlabs.android.dbflow.customhelper

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Description:
 */
@Database(name = HelperDatabase.NAME, version = HelperDatabase.VERSION)
object HelperDatabase {

    const val NAME = "Helper"

    const val VERSION = 1
}
